collect_app/src/main/res/values/strings.xml

```diff
@@ -589,6 +588,8 @@
     <string name="no_columns_to_upload">No columns found in the form to upload.</string>
     <string name="storage_runtime_permission_denied_title">Storage permissions</string>
     <string name="storage_runtime_permission_denied_desc">Without these permissions Collect can\'t access your forms or save answers. Reopen the app when you are ready to grant them.</string>
+    <string name="camera_runtime_permission_denied_title">Camera permission</string>
+    <string name="camera_runtime_permission_denied_desc">Without this permission Collect can\'t access camera. Please try again when you are ready to grant it.</string>
```

Comment:
"access camera" -> "access the camera"

@jd-alexander I think you had perhaps suggested different messages depending on whether or not the question is required? There is a difference -- if the question isn't required then a user can in fact keep using the app without any problems. It's when a question is required that they can't in fact complete the form.

Reply:
Yes, this would be good. The possible verbage could be - Title :  "Camera permission required." Description : "Without this permission, the form can't be submitted." 

It's easier for the user to see this than for them to attempt to submit the form and when it shows the error validation on the question and the permission dialog is presented again and they have to infer that the form can't be submitted.

Reply:
Really? Do you think it's needed? If a question is required a user sees a red asterisk and depending on settings can't even navigate further or in the worst case will be moved back to such a question during completing a form. I think it's enough.

Reply:
I fixed the string but let me know if you are sure we need different strings for required questions.

Reply:
Okay. What we could do is leave it as is since users know of required questions and then if anyone has any complaints then we add it to make things clearer.